style.css

```
@@ -0,0 +1,165 @@
+@charset "UTF-8";
+
+
+
+body{
+        
+    background-color:rgb(253, 251, 251) ;
+    
+    }
+#cabecalho{
+       
+    position: fixed;
+    color:rgba(69, 161, 231, 0.246) ;         
+    font-family: 'Lato';
+    top: 0px;
+    background-color:rgba(119, 12, 48, 0.8);
+    z-index: 2;
+
+    }
+
+img{
+    position: absolute;
+    display: block;
+    top: 430px;
+    left:-150px;
+    size: 60%;
+    border: 1px;
+    border-color: black;
+    padding:1px;
+    width: 500px;
+    
+
+    }
+
+
+#cabecalho{
+
+    width: 1200px;
+
+    }
+
+
+#titulo{
+    font-size: 60px;
+    margin: 8px;
+    font-family: 'Dancing Script', cursive;
+    padding-left: 40px;
+    padding-top: 15px;
+    font-style: italic;
+    z-index:0;
+    color:white;
+    }
+
+
+#cabecabutons li {
+
+    display: inline-block;
+    margin: 8px;
+        
+     }
```

A indentação do colchete deve ser igual à dos atributos!
```suggestion
    }
```